using-mobile-extensions/offer-decisioning/api-reference.md

```diff
@@ -211,6 +211,160 @@ OfferDecisioning.retrievePrefetchedOffers(decisionScopes: [homeDecisionScope]) {
 ## Public classes
 
 {% tabs %}
+{% tab title="Android" %}
+### DecisionScope
+
+This class encapsulates the decision `activityId`, `placementId` and `itemCount`. The `DecisionScope` object can be used to fetch offers from the offer decisioning service if it is enabled in the Experience Edge network. The default value for `itemCount` is 1, if not provided when creating the `DecisionScope` object.
+
+```java
+final public class DecisionScope {
+    final private String activityId;
+    final private String placementId;
+    final private int itemCount;
+
+    public DecisionScope(final String activityId, final String placementId) {...}
+
+    public DecisionScope(final String activityId, final String placementId, final int itemCount) {...}
+
+    public String getActivityId() {
+        return activityId;
+    }
+
+    public String getPlacementId() {
+        return placementId;
+    }
+
+    public int getItemCount() {
+        return this.itemCount;
```

Comment:
Do we need to include the implementation details here? An alternative is to define these APIs like:
```
public int getItemCount() { ... }
```

Same applies for the other classes - methods and constructors.

Reply:
I'm already using ellipsis for the constructor methods here. For the accessors (relatively simple impl), I wanted to indicate the properties that are being returned.